YoCli/Program.cs

```diff
@@ -1,37 +1,34 @@
-﻿using McMaster.Extensions.CommandLineUtils;
-using Microsoft.Extensions.DependencyInjection;
+﻿using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using System;
 using System.Threading.Tasks;
 using YoCli.Commands;
-using YoCli.Services.Implementations;
-using YoCli.Services.Interfaces;
+using YoCli.Commons;
+using YoCli.Data;
+using YoCli.Models;
 
 namespace YoCli
 {
-    public class Program
+    class Program
     {
-        private readonly IConsole _console;
 
-        public Program(IConsole console)
-        {
-            _console = console;
-        }
-
-        public static async Task<int> Main(string[] args)
+        static async Task<int> Main(string[] args)
         {
             var builder = new HostBuilder().ConfigureServices((hostContext, services) =>
             {
-                services.AddSingleton<INoteService, NoteService>();
```

Comment:
Hi @st3no, can we keep the implementation of NoteService and refactor the business logic in the commands (Read, Find, Write, Export, Import) to a another service that implement INoteService ? So you can add some unit tests for the implementation. Thank you very much